Sources/ArcGISToolkit/Components/Augmented Reality/Utilities/GeoTrackingSceneView.swift

```diff
@@ -165,6 +178,7 @@ public struct GeoTrackingSceneView: View {
         cameraController.transformationMatrix = .identity
     }
     
+#if !os(visionOS)
```

Comment:
Could the enclosing methods just be marked as unavailable on visionOS?

Reply:
They could yes but they would still need the compiler directives to compile successfully on visionOS.